lms/static/sass/inline-discussion-rtl.scss

```diff
@@ -0,0 +1,34 @@
+// From lms-main-v1-rtl.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-rtl'; // set the layout for right to left languages
+@import 'base/variables-rtl';
```

Comment:
@e-kolpakov Good question. I based this file on [lms-main-v1-rtl.scss](https://github.com/edx/edx-platform/blob/master/lms/static/sass/lms-main-v1-rtl.scss), which also imports `base/variables-rtl` before `base/variables`. And it turns out that these files don't clash: `variables-rtl.scss` only defines a single variable (`$default-layout-direction`). The corresponding file for LTR is `variables-ltr.scss`, *but this isn't what gets imported in L9 below*. `_variables.scss` defines a bunch of essential variables; without these, this file does not compile. It doesn't touch `$default-layout-direction`, so it's fine to import it after `base/variables-rtl`.